internal/cmd/launcher_test.go

```diff
@@ -479,13 +480,13 @@ func TestIsCustomBuildRequired(t *testing.T) {
 		t.Run(tc.title, func(t *testing.T) {
 			t.Parallel()
 
-			deps := k6deps.Dependencies{}
+			deps := make(map[string]*semver.Constraints)
 			for name, constrain := range tc.deps {
 				dep, err := k6deps.NewDependency(name, constrain)
```

Comment:
```suggestion
			for name, constraint := range tc.deps {
				dep, err := k6deps.NewDependency(name, constraint)
```
Not from your PR so feel free to ignore it or update it in a future PR but in case you apply my other comments, you can apply this one as well.